### ISSUE TITLE
Warn when an RPM does not have compliant license

### ISSUE THREAD

**Issue by xsuchy** [fedora-copr/copr#3290]
The work of migrating packages to [SPDX](https://fedoraproject.org/wiki/Changes/SPDX_Licenses_Phase_4) is almost finished. We can call `license-validate` on the License tag of the resulting packages and check if licenses are compliant with Fedora policy.

When the license is not valid, we can show the user a warning that the license is not allowed. Or it has a typo - this will be common in the early days.

**Comment by befeleme** [comment]
FYI: pyp2spec 0.10.0, released today, will no longer error when encountering a non-compliant license.

**Comment by brianjmurrell** [comment]
> FYI: pyp2spec 0.10.0, released today, will no longer error when encountering a non-compliant license.

Awesome!  What has to happen next for this to be the case in COPR?

**Comment by befeleme** [comment]
First is has to land in all Fedora repositories (soon, up to 7 dayss: https://bodhi.fedoraproject.org/updates/?packages=pyp2spec) 
Then, once this PR https://github.com/fedora-copr/copr/pull/3473 is merged, it'll be automatically pulled and invoked correctly in Copr.

**Comment by FrostyX** [comment]
> Then, once this PR https://github.com/fedora-copr/copr/pull/3473 is merged, it'll be automatically pulled and invoked correctly in Copr.

Yes and no. AFAIK `pyp2spec` is not being automatically updated to the latest version on Copr builders so we will have to update it (easyfix) and we need to get the PR #3473 code into production. Since it is a one-liner, we don't have to wait for a new release and we can hotfix this in production.

Can you please ping us @befeleme once we should do this? :-)
